src/current/v23.2/admission-control.md

```diff
@@ -24,21 +38,39 @@ Admission control can help if your cluster has degraded performance due to the f
+- [Deletes]({% link {{ page.version.version }}/delete-data.md %}) (including deletes initiated by [row-level TTL jobs]({% link {{ page.version.version }}/row-level-ttl.md %}); the [selection queries]({% link {{ page.version.version }}/selection-queries.md %}) performed by TTL jobs are also subject to CPU admission control).
+- [Follower replication work]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
+- [Raft log entries being written to disk]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
+- [Changefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}).
+- [Intent resolution]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-intents).
+
+The following operations are not subject to admission control:
+
+- SQL writes are not subject to admission control on [follower replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) by default, unless those writes occur in transactions that are subject to a Quality of Service (QoS) level as described in [Set quality of service level for a session](#set-quality-of-service-level-for-a-session). In order for writes on follower replicas to be subject to admission control, the setting `default_transaction_quality_of_service=background` must be used.
+
 {{site.data.alerts.callout_info}}
 Admission control is beneficial when overall cluster health is good but some nodes are experiencing overload. If you see these overload scenarios on many nodes in the cluster, that typically means the cluster needs more resources.
 {{site.data.alerts.end}}
 
 ## Enable and disable admission control
 
-To enable and disable admission control, use the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
+Admission control is enabled by default.
+
+To enable or disable admission control, use the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
 
 - `admission.kv.enabled` for work performed by the [KV layer]({% link {{ page.version.version }}/architecture/distribution-layer.md %}).
 - `admission.sql_kv_response.enabled` for work performed in the SQL layer when receiving [KV responses]({% link {{ page.version.version }}/architecture/distribution-layer.md %}).
 - `admission.sql_sql_response.enabled` for work performed in the SQL layer when receiving [DistSQL responses]({% link {{ page.version.version }}/architecture/sql-layer.md %}#distsql).
 
-When you enable admission control Cockroach Labs recommends that you enable it for **all layers**.
-
-Admission control is enabled  by default for all layers.
+When you enable or disable admission control settings for one layer, we recommend that you enable or disable them for **all layers**.
```

Comment:
I know, I know

```suggestion
When you enable or disable admission control settings for one layer, Cockroach Labs recommends that you enable or disable them for **all layers**.
```

Reply:
lol

worst ever addition to the CRL style guide IMO.  but I will adhere to the letter of the law of the land

Reply:
(PS fixed it)